Tests/Factory/PhpredisClientFactoryTest.php

```diff
@@ -1,44 +1,47 @@
 <?php
 
-namespace Snc\RedisBundle\Factory;
+namespace Snc\RedisBundle\Tests\Factory;
 
 use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Snc\RedisBundle\Client\Phpredis\Client;
+use Snc\RedisBundle\Factory\PhpredisClientFactory;
 use Snc\RedisBundle\Logger\RedisLogger;
 
 class PhpredisClientFactoryTest extends TestCase
 {
     protected function setUp()
     {
-        if (!class_exists('\Redis')) {
+        if (!class_exists(\Redis::class)) {
             $this->markTestSkipped(sprintf('The %s requires phpredis extension.', __CLASS__));
         }
 
         if (!@fsockopen('127.0.0.1', 6379)) {
             $this->markTestSkipped(sprintf('The %s requires a redis instance listening on 127.0.0.1:6379.', __CLASS__));
         }
 
-        $this->logger = $this->getMockBuilder('Symfony\Component\HttpKernel\Log\LoggerInterface')->getMock();
+        $this->logger = $this->getMockBuilder(LoggerInterface::class)->getMock();
```

Comment:
That's not the same class as before.

Reply:
It isn't because the previous class was removed: https://github.com/symfony/http-kernel/commit/897836f4af4ecd620804c64e76949e1383ce4330

Reply:
Indeed! Thanks!

I am wondering how it could have worked before :thinking: